projects/plugins/jetpack/extensions/blocks/business-hours/business-hours.php

```diff
@@ -158,7 +158,7 @@ function render( $attributes ) {
 				gmdate( $time_format, $opening ),
 				gmdate( $time_format, $closing )
 			);
-			if ( $key + 1 < count( $day['hours'] ) ) {
+			if ( is_countable( $day['hours'] ) && $key + 1 < count( $day['hours'] ) ) {
```

Comment:
If it's not a numerically-indexed array, other code here is also going to do the wrong thing. Probably we should rework the loop to do something like this
```php
		foreach ( $day['hours'] as $hour ) {
			$opening = strtotime( $hour['opening'] );
			$closing = strtotime( $hour['closing'] );
			if ( ! $opening || ! $closing ) {
				continue;
			}
			if ( $days_hours !== '' ) {
				$days_hours .= ', ';
			}
			$days_hours .= sprintf(
				'%1$s - %2$s',
				gmdate( $time_format, $opening ),
				gmdate( $time_format, $closing )
			);
		}
```